docs/analyzer-rules/AZFW0011.md

```diff
@@ -0,0 +1,32 @@
+# AZFW0011: Invalid binding type
+
+| | Value |
+|-|-|
+| **Rule ID** |AZFW00011|
+| **Category** |[Usage]|
+| **Severity** |Error|
+
+## Cause
+
+This rule is triggered when a function is binding to a non-iterable type for a  blob container path.
+
+## Rule description
+
+For a function that has a BlobInput attribute with a container path, the parameter must be of iterable type such as `IEnumerable<T>`.
+
+For example, if your function has attribute `BlobInput("container")` and it is binding to a `string`, then you will see this error.
```

Comment:
Example: if your function uses `BlobInput("<container path>")` and it is binding to a `string`, this rule will be violated.